python/lsst/ts/Dome/llc_configuration_limits/amcs_configuration_limits.py

```diff
@@ -0,0 +1,85 @@
+from .abstract_llc_configuration_limits import AbstractLlcConfigurationLimits
+import math
+
+DEGREES_TO_RADIANS = math.pi / 180.0
+
+
+class AmcsConfigurationLimits(AbstractLlcConfigurationLimits):
+    """This class holds the limits of the configuration values for the AMCS lower level component. It will
+    validate any set of configuration parameters againt these limits. It will also convert any
+    configuration parameter values expressed in deg/s^n to rad/s^n (with n = 1, 2 or 3).
```

Comment:
Strictly speaking they are not parameters since they cannot be passed as arguments to the constructor. So I added a similar doc string since I agree that a description there is very useful.